tests/parse/test_parse_transcripts.py

```diff
@@ -177,6 +177,20 @@ def test_parse_superdups_fractmatch():
         assert transcript["superdups_fracmatch"] == fract_match
 
 
+def test_parse_cadd(vep_csq_header, vep_csq):
+    """Testing parsing of CADD_PHRED score from VEP-annotated transcripts"""
+
+    # GIVEN a transcript with the CADD score in th CSQ
+    header = [word.upper() for word in vep_csq_header.split("|")]
+    raw_transcripts = [dict(zip(header, entry.split("|"))) for entry in vep_csq.split(",")]
```

Comment:
I would be more keen to know that we actually saw CADD here first? Still, if it wasn't there the test would fail, so that is a pure convenience.

Reply:
I didn't get it 🤔 I picked vep_csq_header and vep_csq because they contain a transcript with CADD score that can be shown to be parsed?

Reply:
It's fine - just a little implicit, compared to the very explicit ones above! 😄 
It is all good as long as no-one pokes with the fixtures - and if someone does the test likely just fails as it should. You could assert that there was actually an appropriate "cadd" in the dict after explicitly making the raw transcript locally though, since that part is not calling a function in the main code? See what I mean?

Reply:
Mmm I'm not sure I get what you mean. There are other 6 tests in that file that use the same fixtures exactly the same way and I took inspiration (ready copy) from them:

- test_parse_functional_annotation
- test_parse_optional_hgnc_annotation
- test_parse_vep_freq_thousand_g_alt
- test_parse_vep_freq_gnomad
- test_parse_vep_freq_mtgnomad
- test_parse_hg38_mane_transcripts

This test, like the other ones, is calling `parse_transcripts` (a function in the main code) to create parsed transcripts.

You want me to check that raw_transcripts contains a cadd value and that corresponds to the same value of the parsed transcripts?

Reply:
I made it a bit more explicit, I think!

Reply:
💯